src/base/registrations/Index.svelte

```diff
@@ -65,16 +83,22 @@
           placeholder=""
           root={config.registrar.domain}
         />
-        {#if label && label.length < 2}
-          <span class="input-info">Please enter a minimum of 2 characters.</span>
-        {:else if label && label.length > 128}
-          <span class="input-info">Please enter a maximum of 128 characters.</span>
+        {#if errors}
+          <div class="input-info">
+            {#each errors as error}
+              <div>{error}</div>
+            {/each}
+          </div>
         {/if}
       </span>
 
-      <button disabled={!label || label.length < 2 || label.length > 128} class="primary register regular" on:click={register}>
-        Check
-      </button>
+        <button
+          disabled={!label || errors.length !== 0}
+          class="primary register regular"
+          on:click={register}
+        >
+          Check
+        </button>
```

Comment:
indent

Reply:
Done in [c5eddde](https://github.com/radicle-dev/radicle-interface/pull/328/commits/c5eddde23f733ad90bf3dd2f94dad2e722a7f4d6)